src/main/java/uk/gov/di/ipv/cri/common/library/domain/personidentity/Address.java

```diff
@@ -153,16 +153,28 @@ public void setValidUntil(LocalDate validUntil) {
 
     @JsonIgnore
     public AddressType getAddressType() {
-        if (Objects.nonNull(this.getValidUntil()) && isPastDate(this.getValidUntil())) {
-            return AddressType.PREVIOUS;
+
+        boolean validFromNotSpecified = (Objects.isNull(this.getValidFrom()));
+        boolean validFromIsSpecified = !validFromNotSpecified;
```

Comment:
I think it's okay to have just the 2 statements mentioned in my original comments and then 
`!validUntilSpecified`
and 
`!validFromSpecified` 
can simply be referred to directly in the conditional statements